server/server_test.go

```diff
@@ -476,6 +499,7 @@ func newPeer() (*ipfslite.Peer, error) {
 		priv,
 		nil,
 		[]multiaddr.Multiaddr{listen},
+		ipfslite.Libp2pOptionsExtra...,
```

Comment:
And added this as well.

Reply:
I'm not sure we'll want all those options... nbd for now though.

Reply:
Cool.